### DISCUSSION TITLE
Was the fix for NEXT-1120 ever released?

### DISCUSSION THREAD

**Discussion by kdmcguire** [vercel/next.js#70620] [App Router] [answered]
We're hitting the same problem described in that issue: our `/api/preview/route.ts` function uses a redirect after the `draftMode().enable();`, so we're not reliably getting the `__prerender_bypass` cookie we need.

It looks like it was merged into `canary` on May 19 2023, but - if I'm reading the canary release descriptions right - it hasn't been included in a release candidate since then.

- https://github.com/vercel/next.js/issues/49237
- https://github.com/vercel/next.js/pull/49965

If it hasn't been included yet, this is our vote for getting it looked at again, ideally in a 14.x.x release 🙂

**Comment by icyJoseph** [comment, marked as answer]
I think it was included here, https://github.com/vercel/next.js/releases/tag/v13.4.3 right? so it should be out already 🤔

**Reply by kdmcguire** [reply]
I think you're right, thank you!
